contracts/contracts/RequestManager.sol

```diff
@@ -268,6 +268,29 @@ contract RequestManager is Ownable, LpWhitelist, RestrictedCalls, Pausable {
             protocolFee(tokenAddress, amount);
     }
 
+    function transferableAmount(
+        uint256 sourceChainId,
+        uint256 targetChainId,
+        address tokenAddress,
+        uint256 amount
+    ) public view returns (uint256) {
+        Token storage token = tokens[tokenAddress];
+        uint256 minFee = minLpFee(sourceChainId, targetChainId, tokenAddress);
+        require(amount > minFee, "Amount not high enough to cover the fees");
+        uint256 transferableAmount = (amount * 1_000_000) /
+            (1_000_000 + token.lpFeePPM + token.protocolFeePPM);
+
+        if (
+            lpFee(sourceChainId, targetChainId, tokenAddress, amount) >= minFee
+        ) {
+            return transferableAmount;
+        }
+
+        return
+            (amount * 1_000_000) /
+            (1_000_000 + token.lpFeePPM + token.protocolFeePPM);
+    }
+
```

Comment:
Do we have a test for this function?

Reply:
will add tests for our view functions similar to minLpFee. I think we didnt have any.